README.md

```diff
@@ -1 +1,39 @@
-# hellomicroverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# microverse-linters-project
+
+> One paragraph statement about the project.
+
+![screenshot](./hellomicproject.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML 
+- CSS
+- Linters
+
+## Live Demo
+
+[Live Demo Link](https://mstanley-me.github.io/microverse-linters-project/)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+## Authors
+
+👤 **Author**
+- Stanley SIMEON
+- GitHub: [@mstanley-me]
+- LinkedIn: [@stanleysimeon]
```

Comment:
- [ ] Your README looks professional. However, why don't you add your correct Github and Linkedin handle? This is to ensure that users can get to you through any of the handles. Note, if you do not have Linkedin yet, that's ok but try adding your correct Github profile link 💯